index.js

```diff
@@ -23,3 +23,123 @@ hbgMenu.addEventListener('click', () => {
+    featureImage: './image/Snapshoot-Portfolio-2.svg',
+    technologies: ['HTML', 'CSS', 'Javascript', 'github', 'ruby', 'Bootstraps'],
+    liveVersion: '',
+    scource: '',
+  },
+  {
+    name: 'Facebook 360',
+    categories: ['Facebook', 'Full Stack Dev', '2015'],
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    featureImage: './image/Snapshoot-Portfolio-3.svg',
+    technologies: ['HTML', 'CSS', 'Javascript', 'github', 'ruby', 'Bootstraps'],
+    liveVersion: '',
+    scource: '',
+  },
+  {
+    name: 'Uber Navigation',
+    categories: ['Uber', 'Lead Developer', '2015'],
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    featureImage: './image/Snapshoot-Portfolio-3.svg',
+    technologies: ['HTML', 'CSS', 'Javascript', 'github', 'ruby', 'Bootstraps'],
+    liveVersion: '',
+    scource: '',
+  },
```

Comment:
### Fourth Project Card 💳
![Capture](https://user-images.githubusercontent.com/93039359/208842468-0c474503-ec9c-4b6c-9635-68e8489eda43.PNG)

### Modal for Fourth Project Card 📳
![modal](https://user-images.githubusercontent.com/93039359/208842572-28fc2a4e-403d-477c-bd2a-96657e1cbbdd.PNG)

- [ ] Now the problem is that if first card and modal for first card is showing correct data according to the array of objects but if i click on second, third or fourth card then modal for these cards don't show the right data 🤔 As you can see the in the above two images that I clicked on the fourth card and popup shows up and the data on the modal is not similar to the card data 🤔 Kindly fix this issue 👍